packages/editor/src/browser/editor-preferences.ts

```diff
@@ -10,36 +10,34 @@ import {
     createPreferenceProxy,
     PreferenceProxy,
     PreferenceService,
+    PreferenceContribution,
+    PreferenceSchema
 } from '@theia/preferences/lib/common';
 
-export const EditorConfigSchema = {
-    "$schema": "http://json-schema.org/draft-04/schema#",
-    "title": "Editor configuration Scheme",
-    "type": "object",
-    "properties": {
+export const editorConfigSchema: PreferenceSchema = {
```

Comment:
I agree, I'm not sure about the Configuration keyword that we currently use i.e EditorConfiguration type and whatnot. I think we should use one of preferences, settings or configuration and use the same everywhere, I feel like it's confusing using PreferenceSchema and DefaultConfiguration whereas PreferenceSchema and DefaultPreferences would make more sense. wdyt?